rpm/qtdeclarative.spec

```diff
@@ -268,6 +285,14 @@ touch .git
 
 make %{?_smp_mflags}
 
+%if 0%{?qt5_release_build}
```

Comment:
Actually, that seems quite convoluted, I don't see a straightforward way to build both debug and release from OBS. I'll just try building it locally and set the flag :)